src/components/ManualFlashingDialog.tsx

```diff
@@ -66,7 +66,6 @@ const ManualFlashingDialog = ({ ...props }: ManualFlashingDialogProps) => {
     >
       <VStack gap={5} width="100%">
         <Text alignSelf="left" width="100%">
-          <FormattedMessage id="connectMB.connectBattery.subtitle" />{" "}
```

Comment:
Small fix: Remove mistakenly added text that says "Disconnect the micro:bit from the computer and connect the battery pack."